### ISSUE TITLE
Emojicons are cut off

### ISSUE THREAD

**Issue by AxeMaster99** [hani-momanii/SuperNova-Emoji#23]
First of all thanks for this nice library. I got a small problem tho: when I have a TextView with only Emojicons inside, they are cut off at the top. When i add some Text, they aren't (see screenshot).
Is there something to solve this problem? Adding a blankspace helps but isn't the best solution. 
Thanks in advance!
<img width="208" alt="issue" src="https://cloud.githubusercontent.com/assets/23356869/24884307/1b54cee4-1e49-11e7-88e7-35d779db5b36.png">

**Comment by hani-momanii** [comment]
can you post the TextView xml code here ?

**Comment by AxeMaster99** [comment]
Of course: 
<img width="735" alt="xml" src="https://cloud.githubusercontent.com/assets/23356869/24898832/75827e0c-1e9e-11e7-837e-3982e230c8e0.png">

**Comment by hani-momanii** [comment]
is the bubble background 9patch ?

**Comment by AxeMaster99** [comment]
Yes it is. But when I delete the background 9 patch and run the application without any design stuff, the problem still occurs and android keeps cutting off the emojicons.

**Comment by VitorHugoDS** [comment]
Hello. I'm having the same issue.
Also using 9patch.
I tried this library for the first time in the 0.0.2 version, and didn't update it for ages. No problems with 9patch back then. Can't tell in which version the bug started to happen.

And also, thanks for this nice library!

**Comment by rohitjagtap1991** [comment]
Hi. I am facing the same issue.
With the updated library version if I show only emojis without text then they cut off!
So I am switching back to version 0.0.2
I am using 9patch as others.
Looking forward to getting a new version with the fix.
Thanks in advance!

**Comment by hamzaahmedkhan** [comment]
app:emojiconAlignment="baseline"
app:emojiconSize="s18"

add these in your attributes. change emoji size according to your textSize.

**Comment by hamzaahmedkhan** [comment]
you are using emoji size s28 but text size s18, so your TextView height is wrap_content and it's wrapping according to s18 size of text.

**Comment by neaenaa** [comment]
You can either use the same size for emojiconSize and textSize, or, add an attribute lineHeight making it the same size as emojiconSize.
